luzer/tests/test_unit.lua

```diff
@@ -120,6 +120,9 @@ ok, err = pcall(fdp.consume_number)
 assert(ok == false)
 assert(err ~= nil)
 
+local n = fdp:consume_number(1, MAX_INT)
```

Comment:
This check looks suspicious and unrelated to this commit (or I don't get it). Also, `MAX_INT` isn't defined yet.